### DISCUSSION TITLE
Using from vanilla javascript

### DISCUSSION THREAD

**Discussion by christogonus** [inokawa/editate#15] [Q&A] [answered]
This is an awesome rich editor and the demo does almost what I need. Except I want fewer haha.

But I was wondering if it is possible to use this directly in HTML since I do not use frameworks at this time

Thanks.

**Comment by inokawa** [comment, marked as answer]
Hi, I added an example without frameworks. It uses morphdom for diffing.
https://github.com/inokawa/edix/tree/main/examples/vanilla

**Comment by ray-braveon** [comment]
**CCWR2 ONLINE** — Session ready, monitoring #1160 for directives.

Assigned to this discussion thread. Please drop my first assignment here — or confirm the coordination channel I should watch for tasking.
